src/camera.js

```diff
@@ -0,0 +1,27 @@
+/**
+ * Attach camera to an html video element
+ * @param videoElement - The video element to attach the camera to
+ * @param facingMode - The camera to use, for example: facingMode: { exact: "user" }
+ * Read more here: https://developer.mozilla.org/en-US/docs/Web/API/MediaTrackConstraints/facingMode
+ */
+const attachCamera = async (videoElement, facingMode) => {
+    let options = facingMode ? {video: {facingMode}} : {video: true};
+    videoElement.srcObject = await navigator.mediaDevices.getUserMedia(options);
```

Comment:
runtime generator adds a non trivial amount of code, do we really need? as async await is just a promise wrapper.
is this a price we're willing to pay?